### DISCUSSION TITLE
Typo in /opt/so/saltstack/default/salt/elasticsearch/files/ingest-dynamic/common

### DISCUSSION THREAD

**Discussion by jimhranicky** [Security-Onion-Solutions/securityonion#6391] [Unsupported Versions] [answered]
Looks like there might be a typo in this file: 

    {source,destination}_geo

instead of

    {source,destination}.geo

Excerpt: 

    -----
    {
        "geoip":  {
                "field": "destination.ip",
                "target_field": "destination_geo",
                "database_file": "GeoLite2-ASN.mmdb",
                "ignore_missing": true,
                "ignore_failure": true,
                "properties": ["ip", "asn", "organization_name", "network"]
        }
    },
    {
        "geoip":  {
                "field": "source.ip",
                "target_field": "source_geo",
                "database_file": "GeoLite2-ASN.mmdb",
                "ignore_missing": true,
                "ignore_failure": true,
                "properties": ["ip", "asn", "organization_name", "network"]
        }
    },
    ----

**Comment by weslambert** [comment, marked as answer]
These technically don't conform to ECS at the moment, but they are named to match what is defined in the template, defined here:
https://github.com/Security-Onion-Solutions/securityonion/blob/master/salt/elasticsearch/templates/so/so-common-template.json.jinja#L134-L167
